pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -110,10 +110,12 @@ func TestAssumePodScheduled(t *testing.T) {
 	}
 
 	tests := []struct {
+		name string
 		pods []*v1.Pod
 
 		wNodeInfo *framework.NodeInfo
 	}{{
+		name: "assumed one pod with resource request and used ports",
```

Comment:
Not sure if the name needs to start with an uppercase. Is there a requirement?

Reply:
Well, either is fine.